components/match2/wikis/valorant/match_group_input_custom.lua

```diff
@@ -114,6 +116,7 @@ function matchFunctions.getTournamentVars(match)
 	match.icon = Logic.emptyOr(match.icon, Variables.varDefault('tournament_icon'))
 	match.icondark = Logic.emptyOr(match.iconDark, Variables.varDefault('tournament_icon_dark'))
 	match.liquipediatier = Logic.emptyOr(match.liquipediatier, Variables.varDefault('tournament_tier'))
+	match.liquipediatiertype = Logic.emptyOr(match.liquipediatiertype, Variables.varDefault('tournament_tiertype'))
```

Comment:
shouldn't it be `tournament_liquipediatiertype`, or are they not yet on standardized infobox league?

Reply:
Not yet standardized infobox league

Reply:
alright